client/ui/client_ui.go

```diff
@@ -716,16 +716,43 @@ func openURL(url string) error {
 
 // checkPIDFile exists and return error, or write new.
 func checkPIDFile() error {
-	pidFile := path.Join(os.TempDir(), "wiretrustee-ui.pid")
+	pidFile := path.Join(os.TempDir(), "netbrid-ui.pid")
 	if piddata, err := os.ReadFile(pidFile); err == nil {
 		if pid, err := strconv.Atoi(string(piddata)); err == nil {
-			if process, err := os.FindProcess(pid); err == nil {
-				if err := process.Signal(syscall.Signal(0)); err == nil {
-					return fmt.Errorf("process already exists: %d", pid)
-				}
+			if pidExists(pid) {
+				return fmt.Errorf("process already exists: %d", pid)
 			}
 		}
 	}
 
 	return os.WriteFile(pidFile, []byte(fmt.Sprintf("%d", os.Getpid())), 0o664) //nolint:gosec
 }
+
+func pidExists(pid int) bool {
+	if runtime.GOOS != "windows" {
+		if process, err := os.FindProcess(pid); err == nil {
+			if err := process.Signal(syscall.Signal(0)); err == nil {
+				return true
+			}
+		}
+	}
+	// Prepare the command
+	cmd := exec.Command("tasklist", "/FI", fmt.Sprintf("PID eq %d", pid))
+
+	// Get the output
+	output, err := cmd.Output()
+	if err != nil {
+		return false
+	}
+
+	// Parse the output
+	scanner := bufio.NewScanner(strings.NewReader(string(output)))
+	for scanner.Scan() {
+		line := scanner.Text()
+		if strings.Contains(line, fmt.Sprintf("%d", pid)) {
+			return true
+		}
+	}
```

Comment:
Can you handle `scanner.Err()` here?

Reply:
If scanner.Err actually returns an error the overall return of the function would be false, the only benefit to handling scanner.Err is to log an issue with the scanning, but it is not like the log is written to disk or anything so ...

Reply:
Right, but if there's an issue that would at least give us a hint when starting the application from CLI